### PR TITLE
Whitelists 2.0

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
Whitelists, take 2

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
adds: Adds whitelists
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/job.dm [hunk 1]
```diff
@@ -163,6 +163,9 @@ SUBSYSTEM_DEF(job)
 		if((job.title in GLOB.faction_whitelist_positions) && (CONFIG_GET(flag/use_role_whitelist))) //If you want a whitelist position, get a whitelist and choose it.
 			continue
 
+		if((job.title in GLOB.faction_whitelist_positions) && (CONFIG_GET(flag/use_role_whitelist))) //If you want an antagonist position, get a whitelist and choose it.
```
Comment (Zero-Percent-Angel) [inline code/controllers/subsystem/job.dm:166]: We don't need the same check twice

**Comment by Zero-Percent-Angel** [comment]
As part of this we should add a admin verb to add to this DB table too

**Comment by Zero-Percent-Angel** [comment]
adding this here so I remember to add the stuff from this to the WL [PR](https://github.com/Afterglow-Ss13/afterglow-ss13/pull/234)

**Comment by Stutternov** [comment]
> adding this here so I remember to add the stuff from this to the WL [PR](https://github.com/Afterglow-Ss13/afterglow-ss13/pull/234)

Give me a DM once that PR is merged to remind me; I’ll update the names and positions. 👍

File: code/controllers/subsystem/job.dm (hunk 1 above)
Comment (Stutternov) [reply code/controllers/subsystem/job.dm:166]: Fixed

**Comment by Stutternov** [comment]
Fixed & Ready

**Comment by Zero-Percent-Angel** [comment]
> As part of this we should add a admin verb to add to this DB table too

I'll merge this when I get a spare minute then add this too afterwards before I do a deploy.

**Pull request merged**